pageserver/src/tenant/timeline.rs

```diff
@@ -3642,12 +3589,54 @@ impl Timeline {
             }
             prev = Some(next_key.next());
         }
+        stats.read_lock_held_compute_holes_micros =
+            stats.read_lock_held_prerequisites_micros.till_now();
         drop(layers);
-        stats.second_read_lock_held_micros = stats.second_read_lock_acquisition_micros.till_now();
+        stats.read_lock_drop_micros = stats.read_lock_held_compute_holes_micros.till_now();
```

Comment:
I thought unlock is always a cheap operation? But it's interesting to observe this value and see what happens...

Reply:
Hm, IDK, something has not run the scheduler code that wakes up other tasks.

But yeah, it's been sub-microsecond in the previous iteration.

I mostly kept it so that all the durations that are for 'when read lock was hel' have the `read_lock_` prefix.